docs/docker/ocpdiag.docker

```diff
@@ -0,0 +1,38 @@
+FROM centos:latest
+
+###############################################################
+# 
+# Sample docker build file to experiment with ocpdiag
+#
+# paste this in ocpdiag.docker
+#
+# docker build --rm ocpdiag.docker centos:ocpdiag
```

Comment:
is this meant as a dev tool or demo?
if former, it would be better to use the more usual docker file structure `<proj name>/Dockerfile`, and that can be expanded on later. and move it somewhere else other than the ./docs folder (/tools or something maybe?)

if latter, id move it do ./docs/demos/ocpdiag.docker and put something else as entrypoint to showcase the demo, rather than a shell

Reply:
I put this as a draft PR rather than shared as a code block in our other communication channels for ease of access by others. My intent was that when I went on to try to build a bare-bones "HelloWorld/SmokeTest" test for the quick start guide, I encountered an error with the bazel build step, I figured the easiest would be to share a reproducible environment for others to review the error.

I'm thinking more of a dev tool but something to discuss, not in its current form.

I used the docs folder as I figured we could have a doc section supporting examples on how to adopt and docker could be one way to onboard folks. 

The entrypoint used is because I'm looking at this more as a runtime environment than a service or demo

Reply:
ok, this wasnt clear to me. there was no message in the PR saying that this is just a request for comments thing and will not actually be merged

Reply:
This was the first time I created a draft PR, https://github.blog/2019-02-14-introducing-draft-pull-requests/ 
it is a bit cryptic, I can see it tagged draft and at the bottom it has a button with "ready for review" but I agree it's not quite intuitive